qa/suites/rbd/persistent-writeback-cache/6-workloads/recovery.yaml

```diff
@@ -6,5 +6,5 @@ tasks:
       image_format: 2
 - exec:
     client.0:
-      - "timeout 10s rbd bench --io-pattern rand --io-type write testimage || true"
+      - "timeout -s SIGKILL 10s rbd bench --io-pattern rand --io-type write testimage || true"
```

Comment:
It would be nice to add an assert that the following `rbd bench` actually exercises the recovery code.  E.g. `rbd status | grep 'Image cache state'` should produce output at this point.